src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -573,6 +573,17 @@ default int getMaxEmotes()
     @Nullable
     VoiceChannel getAfkChannel();
 
+    /**
+     * Provides the {@link net.dv8tion.jda.api.entities.TextChannel TextChannel} that has been set as the channel
+     * <br>If no public updates channel has been set as the AFK channel, this returns {@code null}.
+     * <p>
+     * This value can be modified using {@link GuildManager#setPublicUpdatesChannel(TextChannel)}.
+     *
+     * @return Possibly-null {@link net.dv8tion.jda.api.entities.VoiceChannel VoiceChannel} that is the AFK Channel.
```

Comment:
Shouldnt this, `{@link net.dv8tion.jda.api.entities.VoiceChannel VoiceChannel} that is the AFK Channel.`, become `{@link net.dv8tion.jda.api.entities.TextChannel TextChannel} that is the Public Updates Channel.`,